packages/treats/scripts/config/util/workbox-generator.js

```diff
@@ -0,0 +1,31 @@
+const WorkboxWebpackPlugin = require("workbox-webpack-plugin");
+
+const workboxGenerator = workboxConfig => {
+    let workboxPlugin = [];
+    const {
+        pluginMode,
+        options
+    } = workboxConfig;
+
+    if (!(pluginMode in WorkboxWebpackPlugin)) {
+        console.error(`Workbox error: ${pluginMode} is not a valid workbox plugin mode`);
+        return workboxPlugin;
+    }
+    else {
```

Comment:
Hi, did you mean else`if(pluginMode === "InjectManifest") {` ?

Reply:
I guess I can make it better like this perhaps:
`if (!(pluginMode in WorkboxWebpackPlugin)) {
        console.error("");
        return workboxPlugin;
 } else if (pluginMode === "InjectManifest") {
        const { swSrc } = options;
        
        if (!swSrc || swSrc === "")  { 
            console.error("");
            return workboxPlugin;
}
workboxPlugin = [new WorkboxWebpackPlugin[pluginMode](options)];
return workboxPlugin;
'